plugins/GranularPitchShifter/GranularPitchShifterEffect.h

```diff
@@ -168,7 +168,7 @@ class GranularPitchShifterEffect : public Effect
 	int m_ringBufLength = 0;
 	int m_writePoint = 0;
 	int m_grainCount = 0;
-	int m_timeSinceLastGrain = 0;
```

Comment:
why this arbitrary high value?

Reply:
The plugin spawns a new grain when the time since the last grain surpasses a certain value.  Initializing this to 0 means when the plugin is first run in the project, it has to wait on that timer before it spawns that first grain.  Initializing it to an arbitrary high value guarantees the first grain will spawn right away instead.

Reply:
Ohh, can't you make it in such a way that it spaws a grain at initialization?

Reply:
This behavior is identical.  The timer value already surpasses the value required so it properly spawns a grain the moment the first sample is processed as it should.  Spawning one manually in the constructor would require a ridiculous amount of duplicate code and/or complex performance-harming functions and/or code readability deficits in general, with no upside.

Reply:
Ok then.